diff --git a/loader/wine.inf.in b/loader/wine.inf.in
index 8e6956b3e9f..261a16dc5b9 100644
--- a/loader/wine.inf.in
+++ b/loader/wine.inf.in
@@ -2587,7 +2587,7 @@ StartType=3
 ErrorControl=1
 
 [Services]
-HKLM,%CurrentVersion%\RunServices,"winemenubuilder",2,"%11%\winemenubuilder.exe -a -r"
+HKLM,%CurrentVersion%\RunServices,"winemenubuilder",2,"%11%\winemenubuilder.exe -r"
 HKLM,"System\CurrentControlSet\Services\Eventlog\Application",,16
 HKLM,"System\CurrentControlSet\Services\Eventlog\System","Sources",0x10000,""
