diff --git a/Telegram/SourceFiles/data/components/sponsored_messages.cpp b/Telegram/SourceFiles/data/components/sponsored_messages.cpp
index 4e68421fff..6431b8c5af 100644
--- a/Telegram/SourceFiles/data/components/sponsored_messages.cpp
+++ b/Telegram/SourceFiles/data/components/sponsored_messages.cpp
@@ -239,62 +239,19 @@ void SponsoredMessages::inject(
 }
 
 bool SponsoredMessages::canHaveFor(not_null<History*> history) const {
-	if (history->peer->isChannel()) {
-		return true;
-	} else if (const auto user = history->peer->asUser()) {
-		return user->isBot();
-	}
 	return false;
 }
 
 bool SponsoredMessages::canHaveFor(not_null<HistoryItem*> item) const {
-	return item->history()->peer->isBroadcast()
-		&& item->isRegular();
+        return false;
 }
 
 bool SponsoredMessages::isTopBarFor(not_null<History*> history) const {
-	if (peerIsUser(history->peer->id)) {
-		if (const auto user = history->peer->asUser()) {
-			return user->isBot();
-		}
-	}
 	return false;
 }
 
 void SponsoredMessages::request(not_null<History*> history, Fn<void()> done) {
-	if (!canHaveFor(history)) {
-		return;
-	}
-	auto &request = _requests[history];
-	if (request.requestId || TooEarlyForRequest(request.lastReceived)) {
-		return;
-	}
-	{
-		const auto it = _data.find(history);
-		if (it != end(_data)) {
-			auto &list = it->second;
-			// Don't rebuild currently displayed messages.
-			const auto proj = [](const Entry &e) {
-				return e.item != nullptr;
-			};
-			if (ranges::any_of(list.entries, proj)) {
-				return;
-			}
-		}
-	}
-	request.requestId = _session->api().request(
-		MTPmessages_GetSponsoredMessages(
-			MTP_flags(0),
-			history->peer->input(),
-			MTPint()) // msg_id
-	).done([=](const MTPmessages_sponsoredMessages &result) {
-		parse(history, result);
-		if (done) {
-			done();
-		}
-	}).fail([=] {
-		_requests.remove(history);
-	}).send();
+        return;
 }
 
 void SponsoredMessages::requestForVideo(
@@ -434,20 +391,8 @@ void SponsoredMessages::parseForVideo(
 	});
 }
 
-SponsoredForVideo SponsoredMessages::prepareForVideo(
-		not_null<PeerData*> peer) {
-	const auto i = _dataForVideo.find(peer);
-	if (i == end(_dataForVideo) || i->second.entries.empty()) {
-		return {};
-	}
-	return SponsoredForVideo{
-		.list = i->second.entries | ranges::views::transform(
-			&Entry::sponsored
-		) | ranges::to_vector,
-		.startDelay = i->second.startDelay,
-		.betweenDelay = i->second.betweenDelay,
-		.state = i->second.state,
-	};
+SponsoredForVideo SponsoredMessages::prepareForVideo(not_null<PeerData*> peer) {
+    return {};
 }
 
 FullMsgId SponsoredMessages::fillTopBar(
